src/_rules/color.js

```diff
@@ -1,3 +1,15 @@
 export const opacity = [
   [/^opacity-?(\d+)$/, ([, d], { theme }) => ({ opacity: theme.opacity[d] }), { autocomplete: 'opacity-${opacity}' }],
 ];
+
+export const textColors = [
+  ['text', { color: 'var(--w-text-color)' }],
+  ['text-inverted', { color: 'var(--w-text-color-inverted)' }],
+  ['text-inverted-subtle', { color: 'var(--w-text-color-inverted-subtle)' }],
+  ['text-subtle', { color: 'var(--w-text-color-subtle)' }],
+];
+
+export const bgColors = [
+  ['bg', { color: 'var(--w-background-color)' }],
+  ['bg-subtle', { color: 'var(--w-background-color-subtle)' }],
+];
```

Comment:
No new line at the end. Imagine if we had some way of fixing these things before it hits github :trollface: